### PR TITLE
Fixes/#300 finish e gon100 re heat etrago tasks fail

### LINKED ISSUE #300: finish-eGon100RE-heat_etrago-tasks-fail
The tasks heat_etrago.supply and heat_etrago.store are failing mainly because of lack of input data in scenario_parameters. Additionally, some tasks related to eGon2035 are executed even if the scenario is not required.

### PR DESCRIPTION
**Pull request by CarlosEpia**
Fixes #300. This branch was already tested for DE on the server during the last run.

### CODE REVIEW

File: src/egon/data/datasets/heat_supply/individual_heating.py [hunk 1]
```diff
@@ -384,17 +384,28 @@ def dyn_parallel_tasks_2035():
                     )
                 )
             return tasks
+        def heat_pumps_2035_skipped():
+            print("eGon2035 is not in the list of scenarios. HeatPumps2035 dataset is skipped")
```
Comment (ClaraBuettner) [inline src/egon/data/datasets/heat_supply/individual_heating.py]: Are the tasks for eGon100RE also skipped if the scenario was not selected?

File: src/egon/data/datasets/scenario_parameters/parameters.py [hunk 2]
```diff
@@ -1109,12 +1109,83 @@ def heat(scenario):
...
+        }
+
+        # Insert annualized capital costs
+        parameters["capital_cost"] = {}
+
+        for comp in parameters["overnight_cost"].keys():
+            parameters["capital_cost"][comp] = annualize_capital_costs(
+                parameters["overnight_cost"][comp],
+                parameters["lifetime"][comp],
+                global_settings("eGon2035")["interest_rate"],
```
Comment (ClaraBuettner) [inline src/egon/data/datasets/scenario_parameters/parameters.py]: eGon2035 is used here for eGon100RE

File: src/egon/data/datasets/scenario_parameters/parameters.py [hunk 3]
```diff
@@ -1109,12 +1109,83 @@ def heat(scenario):
...
+                costs, "central air-sourced heat pump", "VOM"
+            ),
+            "central_gas_chp": read_costs(costs, "central gas CHP", "VOM"),
+            "central_gas_boiler": read_costs(
+                costs, "central gas boiler", "VOM"
+            ),
+            "central_resistive_heater": read_costs(
+                costs, "central resistive heater", "VOM"
+            ),
+            "geo_thermal": 2.9,  # Danish Energy Agency
```
Comment (ClaraBuettner) [inline src/egon/data/datasets/scenario_parameters/parameters.py:1131]: For the year 2050 they assume costs for geothermal in district heating of 2.7Euros/MWh_th

File: src/egon/data/datasets/heat_supply/individual_heating.py (hunk 1 above)
Comment (CarlosEpia) [reply src/egon/data/datasets/heat_supply/individual_heating.py]: Good point. Now all the tasks in this script are executed only if the related scenario is required. Fixed in 4a6aa4084f15282dfa89023cf3d9f1ef355c6d29.

File: src/egon/data/datasets/scenario_parameters/parameters.py (hunk 2 above)
Comment (CarlosEpia) [reply src/egon/data/datasets/scenario_parameters/parameters.py]: Right. Fixed in 2b7808f6b410d4bc7b1087e3b0d0370b46a52fc2.

**Review by ClaraBuettner — APPROVED** [review]

**Pull request merged**